layers/+tools/lsp/README.org

```diff
@@ -49,14 +49,15 @@ A number of configuration variables have been exposed via the LSP layer =config.
 Sensible defaults have been provided, however they may all be overridden in your .spacemacs, or dynamically using the bindings added
 under the derived mode t prefix by =(spacemacs/lsp-bind-keys-for-mode mode)=
 
-| Variable name                   | Default | Description                                                                               |
-|---------------------------------+---------+-------------------------------------------------------------------------------------------|
-| =lsp-navigation=                | `both’  | `simple’ or `peek’ to bind only xref OR lsp-ui-peek navigation functions                  |
-| =lsp-ui-remap-xref-keybindings= | nil     | When non-nil, xref key bindings remapped to lsp-ui-peek-find-{definition,references}      |
-| =lsp-ui-doc-enable=             | t       | When non-nil, the documentation overlay is displayed                                      |
-| =lsp-ui-doc-include-signature=  | nil     | When nil, signature omitted from lsp-ui-doc overlay (this is usually redundant)           |
-| =lsp-ui-sideline-enable=        | t       | When non-nil, the symbol information overlay is displayed                                 |
-| =lsp-ui-sideline-show-symbol=   | nil     | When non-nil, the symbol information overlay includes symbol name (redundant for c-modes) |
+| Variable name                   | Default | Description                                                                                   |
+|---------------------------------+---------+-----------------------------------------------------------------------------------------------|
+| =lsp-navigation=                | `both’  | `simple’ or `peek’ to bind only xref OR lsp-ui-peek navigation functions                      |
+| =lsp-ui-remap-xref-keybindings= | nil     | When non-nil, xref key bindings remapped to lsp-ui-peek-find-{definition,references}          |
+| =lsp-ui-doc-enable=             | t       | When non-nil, the documentation overlay is displayed                                          |
+| =lsp-ui-doc-include-signature=  | nil     | When nil, signature omitted from lsp-ui-doc overlay (this is usually redundant)               |
+| =lsp-ui-sideline-enable=        | t       | When non-nil, the symbol information overlay is displayed                                     |
+| =lsp-ui-sideline-show-symbol=   | nil     | When non-nil, the symbol information overlay includes symbol name (redundant for c-modes)     |
+| =lsp-prefer-flymake=            | nil     | When non-nil flymake checker is used. When :none neither flycheck nor flymake checker is user |
```

Comment:
Actual configuration in .spacemacs file is
```emacs-lisp
(lsp :variables lsp-prefer-flymake ':none)
```

Should there be `':none` in documentation or is `:none` ok?